editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-spy-wrapper.spec.browser.tsx

```diff
@@ -304,36 +304,16 @@ describe('Spy Wrapper Template Path Tests', () => {
         },
         "storyboard/scene/app:app-root/inner-div": Object {
           "children": Array [
-            "storyboard/scene/app:app-root/inner-div/card-instance",
+            "storyboard/scene/app:app-root/inner-div/other-card-instance",
           ],
           "name": "div",
           "rootElements": Array [],
         },
-        "storyboard/scene/app:app-root/inner-div/card-instance": Object {
+        "storyboard/scene/app:app-root/inner-div/other-card-instance": Object {
           "children": Array [],
           "name": "Card",
           "rootElements": Array [],
         },
-        "storyboard/scene/app:app-root/inner-div/card-instance:button-instance": Object {
```

Comment:
🚨  this is definitely not right, there's no reason why button-instance and the hi-elements would be gone from here

Reply:
This is missing because the focused element path no longer matches after the uid change

Reply:
Right, I'll have another look at this.

Reply:
I approved the PR assuming this will be fixed :)

Reply:
Fixed up the snapshots in df568e48ce6a3b848d37987eff57315e291b70b6.